app/components/UI/Tokens/index.tsx

```diff
@@ -168,7 +193,7 @@ const Tokens: React.FC<TokensI> = ({ tokens }) => {
     setShowScamWarningModal(false);
   };
 
-  const renderScamWarningIcon = (asset) => {
+  const renderScamWarningIcon = (asset: Asset) => {
```

Comment:
Should we use this instead since it was already used here?
```suggestion
  const renderScamWarningIcon = (asset: TokenI) => {
```